app/ethermint.go

```diff
@@ -138,181 +222,265 @@ func NewEthermintApp(
 	traceStore io.Writer,
 	loadLatest bool,
 	skipUpgradeHeights map[int64]bool,
+	homePath string,
 	invCheckPeriod uint,
-	baseAppOptions ...func(*bam.BaseApp),
+	encodingConfig simappparams.EncodingConfig,
```

Comment:
is this used to specify the type of encoding (amino vs proto) on the ethermint app?